bears/python/PycodestyleBear.py

```diff
@@ -61,3 +70,24 @@ def create_arguments(
         arguments.append(filename)
 
         return arguments
+
+    def process_output(
+            self,
+            output,
+            filename,
+            file,
+            ):
+        output_regex = (r'(?P<line>\d+) (?P<column>\d+) '
+                        r'(?P<message>(?P<origin>\S+).*)')
+        result = re.match(output_regex, output)
+        if not result:
+            return
+        line, column, message, origin = result.groups()
+        yield Result.from_values(
+            origin=self,
+            message=message,
+            file=filename,
+            line=int(line),
+            column=int(column),
+            aspect=Formatting('py'),
```

Comment:
is it possible / sensible to set this to the taste for LineLenght , if the aspect was used to determine the max line length?

Reply:
See the docstring for `Result.aspect`.

> An aspectclass instance which this result is associated to.
>  Note that this should be a leaf of the aspect tree!

Formatting('py') is not the leaf of the aspect tree.

Reply:
Oh, didn't see that one, I'll update it to `LineLength`